src/NuGet.Clients/NuGet.VisualStudio.Common/OutputConsoleLogger.cs

```diff
@@ -104,7 +105,10 @@ public void Log(MessageLevel level, string message, params object[] args)
                     message = string.Format(CultureInfo.CurrentCulture, message, args);
                 }
 
-                NuGetUIThreadHelper.JoinableTaskFactory.Run(() => OutputConsole.WriteLineAsync(message));
+                NuGetUIThreadHelper.JoinableTaskFactory
+                    .Run(async () => (await _outputConsole).WriteLineAsync(message))
+                    .FileAndForget(TelemetryUtility.CreateFileAndForgetEventName(
```

Comment:
This is a sync over async situation, so we don't need the file and forget.

Reply:
Thanks! My bad... I fixed it and added LogAsync overloads.